src/measure.rs

```diff
@@ -79,8 +106,19 @@ async fn setup_subscriber(
     Ok(subscriber)
 }
 
-struct Provider {
-    pub provider_interface: Box<dyn ProviderInterface>,
+fn create_databroker_endpoint(host: String, port: u64) -> Result<Endpoint> {
+    let databroker_address = format!("{}:{}", host, port);
+
+    let endpoint = tonic::transport::Channel::from_shared(databroker_address.clone())
+        .with_context(|| "Failed to parse server url")?;
+    let endpoint = endpoint
+        .initial_stream_window_size(1000 * 3 * 128 * 1024) // 20 MB stream window size
+        .initial_connection_window_size(1000 * 3 * 128 * 1024) // 20 MB connection window size
+        .keep_alive_timeout(Duration::from_secs(1)) // 60 seconds keepalive time
```

Comment:
Duplicate call to keep_alive_timeout()
Comment mismatches actual value.

Are these settings different than the defaults?

Just out of curiosity, if they differ and are more suitable, it would make sense to document it for real clients (not only for databroker-perf).

Reply:
WIll comment out and make it adjustable in the future.

Reply:
Still haven't been removed..